bbp/nse/neurom/default.nix

```diff
@@ -9,33 +7,28 @@ let
 in
 pythonPackages.buildPythonPackage rec {
   pname = "neurom";
+  version = "1.4.9";
   name = "${pname}-${version}";
-  version = "1.4.8";
-
-  src = fetchFromGitHub {
-  owner = "BlueBrain";
-  repo = "NeuroM";
-  rev = "04f48747785265aa7a4f7b0750c1447cae408468";
-  sha256 = "1qi8d2r4nzp41mddng2qxpkz20zvx6khw7g9ly3nkpg6v9anby9z";
-
- };
-
-
-
- propagatedBuildInputs = [
-         pythonPackages.enum34
-         pythonPackages.future
-         pythonPackages.scipy
-         pythonPackages.numpy
-         pythonPackages.pyyaml
-         pythonPackages.tqdm
-         pythonPackages.matplotlib
-         pythonPackages.h5py
-         pythonPackages.pylru
-       ];
-
-    passthru = {
-        pythonDeps = propagatedBuildInputs;
-    };
 
+  src = pythonPackages.fetchPypi {
```

Comment:
switching to `fetchPypi` to ensure correspondence between `pip` and `Nix` versioning